package.json

```diff
@@ -30,49 +30,78 @@
     "url": "https://github.com/BBVAEngineering/ember-cli-markdown-codefences/issues"
   },
   "scripts": {
-    "lint": "eslint ./*.js addon addon-test-support app config lib server test-support tests",
-    "test": "mocha node-tests/*test.js "
+    "lint": "npm run lint:js && npm run lint:hbs",
+    "lint:hbs": "ember-template-lint .",
+    "test": "mocha node-tests/*test.js",
```

Comment:
No coverage in this repo?

Reply:
I can add coverage but I think that there is no point in adding coverage to this kind of tasks.